plugin/forward/forward.go

```diff
@@ -147,11 +148,11 @@ func (f *Forward) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg
 
 			formerr := state.ErrorMessage(dns.RcodeFormatError)
 			w.WriteMsg(formerr)
-			return 0, nil
+			return 0, taperr
```

Comment:
something I always disliked in proxy was the dnstap errorr are now proxy/forward errors, I don't think we want that.

Reply:
actually don't think this matters much, this means it logs.